frontends/web/src/components/transactions/components/address-or-txid.tsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright 2024 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { CopyableInput } from '../../copy/Copy';
+import transactionStyle from '../transactions.module.css';
```

Comment:
With #2781 merged, we should start using absolute imports for imports that are `../` or higher ('`../../` etc..). Thanks 👍